Altis_Life.Altis/stringtable.xml

```diff
@@ -8331,7 +8331,7 @@
             <Spanish></Spanish>
             <Russian></Russian>
             <German></German>
-            <French>Entrer dans le véhicule</French>
+            <French>Obtenir dans le véhicule</French>
```

Comment:
"Obtenir dans le véhicule" = "Obtain in the vehicle", which means nothing in FR/EN. ^^

"Entrer dans le véhicule", or "Monter dans le véhicule" are more accurate for "Get in vehicle"